shellbase-slack/src/main/scala/com/sumologic/shellbase/slack/SlackState.scala

```diff
@@ -28,7 +28,7 @@ trait SlackState {
 
   def slackChannels: List[String] = slackChannel.toList
 
-  def slackOptions: Map[String, String] = Map()
-
   def actorSystem: ActorSystem
+
+  def userNameToBeUsedWhenPosting: String
```

Comment:
I believe this needs a default value to minimize API breakages.

Reply:
OK, adding. Tbh, to explain myself - I didn't consider minimizing of API breakages a goal. I just thought all the users would need to adapt. Obviously, I follow your lead here!